### PR TITLE
Updated with Trie ds in cpp

### PR DESCRIPTION
**Pull request by suraj0223**
Implementation of trie ds in CPP.

### CODE REVIEW

**Comment by suraj0223** [comment]
Is this okay @iamrajiv

File: C++/ds/trie.cpp [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  insert("alex");
+  insert("eric");
+  insert("daniel");
+  insert("zee");
+  insert("zoi");
+  insert("jonathon");
+  insert("darek");
+  insert("amly");
+  string result = search("darek") ? "Found" : "Not Found";
+  std::cout << result << '\n';
```
Comment (iamrajiv) [inline C++/ds/trie.cpp]: @suraj0223 no need to use `std::` since `using namespace std;` is already been used.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: C++/ds/trie.cpp (hunk 1 above)
Comment (suraj0223) [reply C++/ds/trie.cpp]: Resolved !

**Comment by iamrajiv** [comment]
@suraj0223 LGTM! Thanks for the PR.

**Review by iamrajiv — APPROVED** [review]

**Comment by iamrajiv** [comment]
@suraj0223 LGTM! Thanks for the PR.

**Comment by iamrajiv** [comment]
@all-contributors please add @suraj0223 for code

**Comment by iamrajiv** [comment]
@all-contributors please add @suraj0223 for code

**Comment by iamrajiv** [comment]
@all-contributors please add @suraj0223 for code

**Pull request merged**